base/libgit2/libgit2.jl

```diff
@@ -437,7 +447,27 @@ function revcount(repo::GitRepo, fst::AbstractString, snd::AbstractString)
     return (fc-1, sc-1)
 end
 
-""" git merge [--ff-only] [<committish> | FETCH_HEAD] """
+"""
+    merge!(repo::GitRepo; committish::AbstractString="", branch::AbstractString="", fastforward::Bool=false, merge_opts::MergeOptions=MergeOptions(), checkout_opts::CheckoutOptions=CheckoutOptions()) -> Bool
```

Comment:
maybe just kwargs... in the signature and add the types and defaults in the bulleted list below